step-templates/delete-azure-resource-group.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Can we rename file to azure-delete-resource-group to group it with the others?

Reply:
Fixed